controllers/identity.controller.ts

```diff
@@ -0,0 +1,94 @@
+import { reciterConfig } from '../config/local'
+import models from '../src/db/sequelize';
+
+export async function getIdentity(uid: string | string[])  {
+    
+   return fetch(`${reciterConfig.reciter.reciterIdentityEndpoints.identityByUid}` +'?uid=' + uid, {
```

Comment:
just a note. Dont have to change anything but we should in the future check if these env variables are actually available otherwise probally return a 401 unauthorized error

Reply:
sure.

Reply:
[302](@JBrutus3: Check for environment variables before using them. #302)